Dockerfile

```diff
@@ -3,7 +3,7 @@ RUN mkdir -p /go/src/github.com/openfaas-incubator/vcenter-connector
 WORKDIR /go/src/github.com/openfaas-incubator/vcenter-connector
 
 COPY vendor     vendor
-COPY main.go    .
+COPY *.go    ./
```

Comment:
I'd rather you didn't change this. Specifying individual files allows for better caching.